src/main/java/mate/academy/hibernate/relations/lib/Dao.java

```diff
@@ -0,0 +1,8 @@
+package mate.academy.hibernate.relations.lib;
```

Comment:
i don't think it was necessary to create this package and classes in it. we should've practice another way to inject dependencies